plugins/about/DevMode.qml

```diff
@@ -2,8 +2,7 @@
  * This file is part of system-settings
  *
  * Copyright (C) 2014 Canonical Ltd.
- *
- * Contact: Oliver Grawert <ogra@canonical.com>
+ * Copyright (C) 2020 Ubports Foundation <developers@ubports.com>
```

Comment:
Exactly speaking, it is capitalized UBports

Reply:
Done!